db/seeds/development/7_groups.rb

```diff
@@ -0,0 +1,18 @@
+@alphabet = ("a".."z").to_a
+
+def generate_group(conference_id)
+  new_group = Group.create(
+    name: @alphabet.shift,
+    conference_id: conference_id,
+  )
+end
+
+(1..5).each do
+  generate_group(1)
```

Comment:
Instead of passing in `1`, let's do a `conference = Conference.first` somewhere and in the `generate_group` helper do `conference: conference` (you don't have to do `..._id`)!